docs/src/components/DrawerDocs.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Drawer from '../markdown/Drawer.mdx';
```

Comment:
Should we rename this to be `DrawerMarkdown ` as to not confuse it with the actual `Drawer` component?